docker/run-test

```diff
@@ -7,10 +7,15 @@ ROOT_DIR=$(cd $(dirname $0)/../sources; pwd -P)
 SCRIPTS_DIR=$(cd $(dirname $0)/../local; pwd -P)
 SOURCE_IMAGE=${1:-centos:7}
 
+if [[ -d /mnt/mtr_disk_dir ]] && [[ ${ENABLE_TESTS_ON_CI_FS} == "yes" ]] && [[ "${CMAKE_BUILD_TYPE}" == "RelWithDebInfo" ]]; then
```

Comment:
Does it mean that CI tests will be executed only for release builds?

Reply:
As pipeline launches, there will be two builds on the same worker, creating an interference between them

I decided to point to a label, so that only one is going, will investigate on it to find a better solution